api/_lib/FanoutGraphqlAppLambdaCallbackTest.test.ts

```diff
@@ -81,7 +81,7 @@ export class FanoutGraphqlAppLambdaCallbackTest {
     Expect(typeof handler).toBe("function");
     await LambdaTester(PulumiCallbackForLambdaTester(handler))
       .event(event)
-      .expectResult(result => {
+      .expectResult((result: any) => {
```

Comment:
Why type this as `any`?

Reply:
TypeScript was complaining about implicit any so I had to add the type here.

Reply:
Got it. idk why my `tsc` wasn't showing me this! Thanks for fixing.